### PR TITLE
More wiki call that we missed

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

**Comment by jcristau** [comment]
Rebasing this on #108 should fix the failing test.

File: auto_nag/common.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    return json.load(urlopen(url))
+
+
+def getVersion(jsonVersion, key):
+    # In X.Y, we just need X
+    version = jsonVersion[key].split(".")
+    return version[0]
+
+
+def getCurrentVersion():
```
Comment (jcristau) [inline auto_nag/common.py]: get_current_versions(), since it doesn't just return one?

File: auto_nag/common.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        esr_version = esr_next_version
+    else:
+        # We are in a cycle where we don't have esr_next
+        # For example, with 52.6, esr_next doesn't exist
+        # But it will exist, once 60 is released
+        # esr_next will be 60
+        esr_version = getVersion(jsonContent, "FIREFOX_ESR")
+
+    return {"central": getVersion(jsonContent, "FIREFOX_NIGHTLY"),
+            "aurora": getVersion(jsonContent, "FIREFOX_AURORA"),
```
Comment (jcristau) [inline auto_nag/common.py]: can we get rid of aurora?  it doesn't actually worse, so this is a bit of a footgun.

File: auto_nag/tests/test_query_creator.py [hunk 3]
```diff
@@ -16,8 +14,8 @@ def test_1_getTemplateValue(self):
         Tests for getTemplateValue,
         Expecting a VERSION number
         """
-        url = "https://wiki.mozilla.org/Template:BETA_VERSION"
-        beta_version = getTemplateValue(url)
```
Comment (jcristau) [inline auto_nag/tests/test_query_creator.py]: this is a test for getTemplateValue, we shouldn't silently make it test something else (and we should keep a test for getTemplateValue as long as that exists and is used for anything)

File: auto_nag/common.py (hunk 2 above)
Comment (jcristau) [reply auto_nag/common.py]: s/worse/work/

File: auto_nag/tests/test_query_creator.py [hunk 4]
```diff
@@ -1,16 +1,25 @@
...
+    def test_0_get_current_versions(self):
+        versions = get_current_versions()
+        nightly_version = versions["central"]
+        assert type(int(nightly_version)) is not type(int)
+        beta_version = versions["beta"]
+        assert type(int(beta_version)) is not type(int)
+        release_version = versions["release"]
+        assert type(int(release_version)) is not type(int)
+        esr_version = versions["esr"]
+        assert type(int(esr_version)) is not type(int)
```
Comment (jcristau) [inline auto_nag/tests/test_query_creator.py]: I'm having a hard time figuring out what this is actually testing.

type(int(foo)) is going to be int (or long.  or it's going to raise).  type(int) is type.  Those are never going to be the same thing.  What am I missing?

Comment (sylvestre) [reply auto_nag/tests/test_query_creator.py]: that the keys exists and they are returning something :)
what would you prefer?

**Pull request merged**